src/coreclr/hosts/corerun/corerun.cpp

```diff
@@ -74,7 +74,8 @@ static void wait_for_debugger()
     }
     else if (state == pal::debugger_state_t::not_attached)
```

Comment:
unrelated to this PR, but this function in corerun.hpp is duplicated with less portable implementation than the one being used in rest of the coreclr https://github.com/dotnet/runtime/blob/c1e7ca823f45082c3459e3211ece37456b2604dd/src/coreclr/pal/inc/pal.h#L291 we could move the implementation to common place, e.g. https://github.com/dotnet/runtime/blob/89313ca06932bdb69c53965ec1e37b67f920ee95/src/native/common/getexepath.h#L26 and then use it from there for DRY'ness.

Reply:
Moving it to a shared location would be nice. I may look into that in a future PR. Thanks.